internal/kafka/command_broker_task_list.go

```diff
@@ -29,20 +29,20 @@ type brokerTaskData struct {
 	ErrorMessage      string                     `human:"Error Message,omitempty" serialized:"error_message,omitempty"`
 }
 
-func (c *brokerCommand) newGetTasksCommand() *cobra.Command {
+func (c *brokerCommand) newTaskListCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "get-tasks [id]",
+		Use:   "list [id]",
 		Short: "List broker tasks.",
 		Args:  cobra.MaximumNArgs(1),
-		RunE:  c.getTasks,
+		RunE:  c.taskList,
 		Example: examples.BuildExampleString(
 			examples.Example{
 				Text: "List remove-broker tasks for broker 1.",
-				Code: "confluent kafka broker get-tasks 1 --task-type remove-broker",
+				Code: "confluent kafka broker task list 1 --task-type remove-broker",
```

Comment:
Require `--broker`?
```suggestion
				Code: "confluent kafka broker task list --broker 1 --task-type remove-broker",
```

Reply:
This command takes the broker id as an argument.